### ISSUE TITLE
Contributor's Guide Prerequisites: Unclear Versioning

### ISSUE THREAD

**Issue by JBallin** [freeCodeCamp/freeCodeCamp#17268]
[Setup/Prerequisites (Contributor’s Guide)](https://github.com/freeCodeCamp/freeCodeCamp/blob/staging/CONTRIBUTING.md#prerequisites):

> ## Setup
> ### Prerequisites
> 
> | Prerequisite                                | Version |
> | ------------------------------------------- | ------- |
> | [MongoDB Community Server](https://docs.mongodb.com/manual/administration/install-community/) | `~ ^3`  |
> | [MailHog](https://github.com/mailhog/MailHog) | `~ ^1` |
> | [Node.js](http://nodejs.org)                | `~ ^8.9.3`  |
> | npm (comes with Node)                       | `~ ^5`  |
> 
> > _Updating to the latest releases is recommended_.

### Problem
**TLDR**: It's unclear which versions are needed. I ran into problems specifically because I was using the latest versions of node and npm. I fixed this using nvm to get the LTS version of node.

I ran into `npm install` issues and @raisedadead informed me that I should be using the "Long Term Support" versions of node and npm (I have the latest installed). You'll see LTS on the left on https://nodejs.org.

That's really confusing for a couple reasons:
1. The `~ ^` suggest 8.9.3 (node) or 5 (npm) and above (although they mean something else), and I wouldn't expect a beginner to understand what they truly mean. Additionally, I can't find any resource explaining what those two characters mean together - I only see their meaning in isolation.
2. The last line says "...latest releases is recommended" which is really misleading, I believe you mean the latest LTS version.

### Needed
1. Clarify `~ ^` symbols
2. Give clear instructions on node so people don't get stuck
3. Clarify latest *LTS* version

**Comment by JBallin** [comment]
I'll work on this and submit a PR shortly

**Comment by Nirajn2311** [comment]
@JBallin what’s the status

**Comment by JBallin** [comment]
@Nirajn2311 haven't had a chance to finish but I have this started. Will attempt to submit a PR this weekend.

**Comment by JBallin** [comment]
Fixed in #18246
